aws/api-servers.tf

```diff
@@ -31,11 +31,10 @@ resource "aws_elb" "api-ext" {
     lb_protocol = "http"
   }
   listener {
-    instance_port = 8080
-    instance_protocol = "http"
+    instance_port = 443
+    instance_protocol = "tcp"
```

Comment:
Should we add an additional `listener` to do HTTP *and* HTTPS? That would match what we're doing for GCE and we could cleanup both as part of the "shut down insecure Tsuru API" story.

Sorry, I missed this the first time.